### PR TITLE
[TTAHUB-3597] QA Dashboard FEI root causes looks to not be counting or filtering correctly

### PR DESCRIPTION
**Pull request by GarrettEHill**
## Description of change

Distinct is not working as expected, switching to group by

## How to test
run qa dashboard

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3597

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/fetchers/ssdi.js [hunk 1]
```diff
@@ -95,9 +95,11 @@ export const getSelfServiceData = async (filterName, filters, dataSetSelection =
...
-    throw new Error('Error fetching self service data');
-  }
-  return response.json();
+
+  return get(urlToUse).then((response) => {
+    if (!response.ok) {
+      throw new Error('Error fetching self service data');
+    }
+    return response.json();
+  });
```
Comment (thewatermethod) [inline frontend/src/fetchers/ssdi.js:104]: what is the reason for this change? this code is functionally identical

**Comment by thewatermethod** [comment]
Can we update how to test section what is the problem we are validating this solves? I can't tell by opening the QA dash

**Review by hardwarehuman — APPROVED** [review]
This looks right, but with so much linter change in there it's hard to be sure I looked at everything.

File: frontend/src/fetchers/ssdi.js (hunk 1 above)
Comment (GarrettEHill) [reply frontend/src/fetchers/ssdi.js:104]: it returns a promise now, it did not before

Comment (GarrettEHill) [reply frontend/src/fetchers/ssdi.js:104]: so now the requests are actually performed in parallel

Comment (thewatermethod) [reply frontend/src/fetchers/ssdi.js:104]: It doesn't matter to me if you change it back or not I was just trying to understand why we are breaking convention here

Comment (GarrettEHill) [reply frontend/src/fetchers/ssdi.js:104]: but it was still waiting for get(urlToUse) to comeback which is what was taking the time before returning the promise
